zppy/templates/default.ini

```diff
@@ -89,6 +89,10 @@ ts_fmt = string(default="ts_only")
   tpd = integer(default=None)
   ts_fmt = string(default=None)
 
+  [[atm_monthly_glb]]
+  # No U
+  vars = string(default="FSNTOA,FLUT,FSNT,FLNT,FSNS,FLNS,SHFLX,QFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,CLDTOT,CLDHGH,CLDMED,CLDLOW")
```

Comment:
@golaz The problem with this approach is that this automatically adds a `atm_monthly_glb` `ts` subtask, even if one isn't requested. The other alternative is to remove `U` in the `ts.py` file, but that would mean the user has no way to add `U` if desired.